.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,53 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.224.2/containers/jekyll
+{
+  "name": "Docs",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      // Update 'VARIANT' to pick a Debian OS version: bullseye, buster
+      // Use bullseye when on local arm64/Apple Silicon.
+      "VARIANT": "bullseye",
+      // Enable Node.js: pick the latest LTS version
+      "NODE_VERSION": "lts/*"
+    }
+  },
+
+  // Set *default* container specific settings.json values on container create.
+
+  // Add the IDs of extensions you want installed when the container is created.
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "GitHub.vscode-pull-request-github",
+        "DavidAnson.vscode-markdownlint",
+        "valentjn.vscode-ltex"
```

Comment:
Just curious, any specific reason for these extensions?

Reply:
This is my personal quality of life selection.
- The first allows opening and reviewing pull requests without leaving the editor. It also shows PR messages right inside the code. To my knowledge one of the default extensions installed in github.dev as well. (Try pressing `.` in any github repo!)
- markdownlint is a great extension helping with creating valid and good looking markdown content. 
- LTex is a linter for grammar and spelling - and does a pretty good job. If we can help reduce the amount of spelling errors in the internet, why not? :D

Reply:
I'm wary of bikeshedding, but basically I (generally) like to avoid relying on third-party extensions if they're not strictly necessary. I'm particularly wary of any grammar and spelling extensions, since they tend to have *many* false positives, particularly for things related to software tooling - which is a big chunk of our userbase.

However, I trust your judgement; if you think these are good defaults for users, we can add them and see how users respond!